app/views/finders/show.atom.builder

```diff
@@ -0,0 +1,21 @@
+atom_feed do |feed|
+  feed.title(@finder.name)
+
+  dates = []
+  @results.results.map(&:metadata).each do |r|
```

Comment:
While that is correct, it has just me realise that the date we get back attached to the document isn't the same one that Rummager is sorting them by.